app/controllers/application_controller.rb

```diff
@@ -14,6 +15,16 @@ def root
 
   private
 
+  def authenticate
+    sign_in(model_by_param(User, :email))
+  rescue
```

Comment:
This is because the function worked for all actin, so the tests dropped for post action#create, etc
now i fix it